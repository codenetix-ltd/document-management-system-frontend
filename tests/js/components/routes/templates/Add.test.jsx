import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import TemplateAdd from 'Components/routes/templates/Add';

describe('Templates Add', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<TemplateAdd />);
  });

  it('should render correctly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
