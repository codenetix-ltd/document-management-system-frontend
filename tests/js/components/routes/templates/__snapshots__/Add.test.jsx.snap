// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Templates Add should render correctly 1`] = `
<div>
  <ContentHeader
    breadcrumbs={
      Array [
        Object {
          "iconCls": "fa fa-copy",
          "pageLink": "/templates/list",
          "pageName": "Templates",
        },
        Object {
          "iconCls": "fa fa-plus",
          "pageLink": "/templates",
          "pageName": "Template create",
        },
      ]
    }
    title="Template create"
  />
  <ContentWrapper
    boxClass="box-info"
    noBox={false}
  >
    <div
      className="box-header with-border"
    >
      <h3
        className="box-title"
      >
        Template data
      </h3>
    </div>
    <Connect(TemplateForm)
      match={
        Object {
          "params": Object {},
        }
      }
      validate={[Function]}
    />
  </ContentWrapper>
</div>
`;
